<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-QTFMA3N2" failures="1" tests="5" name="Personal Loan" time="0.072" errors="0" timestamp="2025-07-03T16:40:29 IST">
  <testcase classname="test.day1" name="demo" time="0.016">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day1.demo(day1.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
  <testcase classname="test.day2" name="ploan" time="0.002"/>
  <testcase classname="test.day4" name="LoginAPIhomeLoan" time="0.002"/>
  <testcase classname="test.day4" name="MobileloginHomeLoan" time="0.002"/>
  <testcase classname="test.day4" name="WebloginHomeLoan" time="0.002"/>
  <testcase classname="test.day1" name="demo2" time="0.0">
    <ignored/>
  </testcase> <!-- demo2 -->
</testsuite> <!-- Personal Loan -->
